src/bun.js/bindings/exports.zig

```diff
@@ -1852,6 +1852,10 @@ pub const ZigConsoleClient = struct {
                     if (key.eqlComptime("constructor")) return;
                     if (key.eqlComptime("call")) return;
 
+                    if (value.jsType().isFunction()) {
```

Comment:
What happens if it's a function defined on the instance but not on the prototype? 

```js
class FooBar {
  constructor() {
    this.abc = () => {};
  }
  method() {
    return 42;
  }
}
```

In node, this logs
```js
FooBar { abc: [Function (anonymous)] }
```